src/components/tables/PoolsTable/PoolsTable.vue

```diff
@@ -1,3 +1,186 @@
+<script setup lang="ts">
+import { ref } from 'vue';
+import { useRouter } from 'vue-router';
+import { useI18n } from 'vue-i18n';
+
+import {
+  DecoratedPoolWithShares,
+  PoolToken
+} from '@/services/balancer/subgraph/types';
+
+import { getAddress } from '@ethersproject/address';
+
+import { POOLS } from '@/constants/pools';
+
+import useNumbers from '@/composables/useNumbers';
+import useFathom from '@/composables/useFathom';
+import useDarkMode from '@/composables/useDarkMode';
+import useBreakpoints from '@/composables/useBreakpoints';
+import {
+  isStableLike,
+  isStablePhantom,
+  isMigratablePool
+} from '@/composables/usePool';
+import { networkId } from '@/composables/useNetwork';
+
+import LiquidityAPRTooltip from '@/components/tooltips/LiquidityAPRTooltip.vue';
+import { ColumnDefinition } from '@/components/_global/BalTable/BalTable.vue';
+
+import TokenPills from './TokenPills/TokenPills.vue';
+
+/**
+ * TYPES
+ */
+type Props = {
+  data?: DecoratedPoolWithShares[];
+  isLoading?: boolean;
+  isLoadingMore?: boolean;
+  showPoolShares?: boolean;
+  noPoolsLabel?: string;
+  isPaginated?: boolean;
+  selectedTokens?: string[];
+  showMigrationColumn?: boolean;
+};
+
+/**
+ * PROPS & EMITS
+ */
+
+const props = withDefaults(defineProps<Props>(), {
```

Comment:
There's no default for isLoading

Reply:
I think if you don't set the default it's `falsy` anyway so it's not critical.